spring-boot-hello-world/src/main/java/com/jalasoft/convert/middleware/AudioControllerMiddleware.java

```diff
@@ -9,7 +9,7 @@
 
 package com.jalasoft.convert.middleware;
 
-import com.jalasoft.convert.common.exception.FileNotFoundException;
+import com.jalasoft.convert.common.exception.MiddlewareException;
 import com.jalasoft.convert.common.logger.At18Logger;
 import javax.servlet.*;
```

Comment:
Hello Mauricio there are still Imports with "*", please try to remove them with the correct imports